Sources/DispatchAsync/DispatchTime.swift

```diff
@@ -0,0 +1,72 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift.org project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift.org project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@available(macOS 13, *)
+public typealias DispatchTime = ContinuousClock.Instant
+
+/// The very first time someone tries to reference a `uptimeNanoseconds` or a similar
+/// function that references a beginning point, this variable will be initialized as a beginning
+/// reference point. This guarantees that all calls to `uptimeNanoseconds` or similar
+/// will be 0 or greater.
+///
+/// By design, it is not possible to related `ContinuousClock.Instant` to
+/// `ProcessInfo.processInfo.systemUptime`, and even if one devised such
+/// a mechanism, it would open the door for fingerprinting. It's best to let the concept
+/// of uptime be relative to previous uptime calls.
+@available(macOS 13, *)
+private let uptimeBeginning: DispatchTime = DispatchTime.now()
+
+@available(macOS 13, *)
+extension DispatchTime {
+    public static func now() -> DispatchTime {
+        now
+    }
+
+    public var uptimeNanoseconds: UInt64 {
+        let beginning = uptimeBeginning
+        let rightNow = DispatchTime.now()
```

Comment:
Oops. I wrote a bad bug here. Just found it with some new test code I'm writing. For some reason I was thinking of this function as a static variable, but it's not, it's a member variable. So we should NOT be calling DispatchTime.now() in here.

Reply:
Fixed